runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -99,7 +138,40 @@ private void processInfobox()
 		{
 			removeKingdomInfobox();
 		}
+	}
+
+	private void setKnownValues()
+	{
+		if (client.getGameState() == GameState.LOGGED_IN && hasCompletedQuest() && isInKingdom())
+		{
+			configManager.setConfiguration(CONFIG_GROUP_NAME, CONFIG_LAST_CHANGED, Instant.now().getEpochSecond());
```

Comment:
These should be added as hidden config options that can be then be used by the overlay and your code

Reply:
Do you mind pointing me towards an example of this?

Reply:
https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerConfig.java#L131

Reply:
Thank you. Added KingdomConfig.
Is it a known issue that ConfigManager can't store longs?

Reply:
store the Instant see `ConfigManager::stringToObject`

Reply:
Changed to store lastChanged as java.time.Instant instead of int representation of decaseconds epoch